src/__experimental__/components/radio-button/radio-button-group.component.js

```diff
@@ -73,7 +73,7 @@ const RadioButtonGroup = (props) => {
     >
       <RadioButtonGroupStyle
         data-component="radio-button-group"
-        role="group"
+        role="radio"
```

Comment:
Are you sure this is the right role for this? My understanding is that the `role="radio"` attribute will identify this as as an ARIA radio button. I think this needs to be `role="radiogroup"` as it's the container for the radio buttons, but I'm not sure of the impact of moving the role from the `FieldSet` in this component is tbh

Reply:
That's a valid point. `radio` will give the impression of a single radio whereas this actually a radio group so should be `radiogroup`. I'll try moving the role from the `FieldSet` and if axe throws any errors.